### ISSUE TITLE
suggest to bump srsly version to 1.0.2 in requirements.txt

### ISSUE THREAD

**Issue by davilayang** [heartexlabs/label-studio#454]
Hi, 
Thanks for the wonderful project.

While annotating with `label-studio` for NER tasks, I have set up backend service with `spaCy` for pre-labelling.    
However, `spaCy` requires `srsly>=1.0.2,<1.1.0` but `label-studio` requires `srsly==0.2.0`. 

I was able to set up the services with source codes by changing `label-studio` to use `1.0.2`, and it works fine for my use case. 
So here to suggest bumping the version to `1.0.2`? That would help with easier installation through PyPI. 

+ [spaCy/requirements.txt](https://github.com/explosion/spaCy/blob/master/requirements.txt)

**Comment by makseq** [comment]
Will it be ok If I make `srsly>=0.2.0`?

**Comment by makseq** [comment]
Can you check release 0.7.5.post1, it's fixed there.

**Comment by davilayang** [comment]
Just checked with `pip install label-studio==0.7.5.post1` and `pip install spacy`. 
The requirement conflict is still there, 
```bash
label-studio 0.7.5.post1 requires srsly==0.2.0, but you'll have srsly 1.0.2 which is incompatible.
```
change to `srsly>=0.2.0` should also work.

**Comment by makseq** [comment]
Sorry, I was wrong - it's 0.7.5.post2

**Comment by davilayang** [comment]
Yes, the version conflict is resolved with `0.7.5.post2`. Thanks.!
